tests/README.rst

```diff
@@ -0,0 +1,45 @@
+Testing Blivet
+==============
+
+Note: The test suite documented here is available only from the git repository
+not as part of any installable packages.
+
+In order to execute blivet's test suite you need root access on the system.
+From inside the source directory execute the commands::
+
+    make test
+    make check
```

Comment:
I never use the check target, because I have to extend my PYTHONPATH with a bunch of directories, including pocketlint. But YMMV.

Also, you absolutely don't need to be root to run pylint.